File: backend-Java/src/main/java/pl/kacste/patient/Patient.java
package pl.kacste.patient;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Entity;
import javax.persistence.Id;

@Entity
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Patient {
    @Id
    private String identityNumber;

    private String firstName;
    private String lastName;
    private String phoneNumber;
    private String postalCode;
    private String city;
    private String street;
}

File: backend-C#/DoctorsOffice/Patients/Patient.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace DoctorsOffice.Patients
{
    public class Patient
    {
        [Key]
        public string IdentityNumber { get; set; }
        
        
        
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string PostalCode { get; set; }
        public string City { get; set; }
        public string Street { get; set; }
    }
}